tests/integration/integration.go

```diff
@@ -26,12 +27,20 @@ var (
 	C_DENOM = "ccoin"
 )
 
-func NewCleanChain() *KeeperTestSuite {
-	return NewCleanChainAtTime(time.Now().Unix())
+type SuiteOpt = func(*KeeperTestSuite)
+
+func NewCleanChain(suiteOpts ...SuiteOpt) *KeeperTestSuite {
+	return NewCleanChainAtTime(time.Now().Unix(), suiteOpts...)
 }
 
-func NewCleanChainAtTime(startTime int64) *KeeperTestSuite {
-	s := KeeperTestSuite{}
+func NewCleanChainAtTime(startTime int64, suiteOpts ...SuiteOpt) *KeeperTestSuite {
+	s := KeeperTestSuite{
+		HandleHooks: make(map[warpTypes.HypTokenType]middleware.HandleHook),
+	}
+
+	for _, opt := range suiteOpts {
+		opt(&s)
+	}
```

Comment:
nit: missing a newline here

```suggestion
	}

```